src/ol_openedx_canvas_integration/README.rst

```diff
@@ -9,6 +9,14 @@ We had to make some changes to edx-platform itself in order to add the "Canvas"
 
 The ``edx-platform`` branch/tag you're using must include one of the below commit for ``ol-openedx-canvas-integration`` plugin to work properly:
 
+**For "Sumac" or more recent release of edX platform, you should cherry-pick below commit:**
+
+TBA
+
+**For "Quince" or more recent release of edX platform, you should cherry-pick below commit:**
+
+https://github.com/mitodl/edx-platform/commit/7a2edd5d29ead6845cb33d2001746207cf696383
+
 **For "Nutmeg" or more recent release of edX platform, you should cherry-pick below commit:**
```

Comment:
I commented because `Nutmeg and prior` means Nutmeg, Quince, and Sumac. These releases have specific commits/versions specified, so I think we can mention a single release in this.